prow/plugins/label/label.go

```diff
@@ -51,7 +65,45 @@ type githubClient interface {
 }
 
 func handleIssueComment(pc plugins.PluginClient, ic github.IssueCommentEvent) error {
-	return handle(pc.GitHubClient, pc.Logger, ic)
+	ae := assignEvent{
+		action:  ic.Action,
+		body:    ic.Comment.Body,
+		login:   ic.Comment.User.Login,
+		org:     ic.Repo.Owner.Login,
+		repo:    ic.Repo.Name,
+		url:     ic.Comment.HTMLURL,
+		number:  ic.Issue.Number,
+		issue:   ic.Issue,
+		comment: ic.Comment,
```

Comment:
"url" and "issue" are not being used. will you suggest removing them? i am following assign plugin template (assign.go).